packages/dev/core/src/Gizmos/scaleGizmo.ts

```diff
@@ -267,6 +271,23 @@ export class ScaleGizmo extends Gizmo implements IScaleGizmo {
         return this._anchorPoint;
     }
 
+    /**
+     * posture that the gizmo will be display
+     * When set null, default value will be used (Quaternion(0, 0, 0, 1))
+     */
+    public get customRotationQuaternion(): Nullable<Quaternion> {
+        return this._customRotationQuaternion;
+    }
+
+    public set customRotationQuaternion(customRotationQuaternion: Nullable<Quaternion>) {
+        this._customRotationQuaternion = customRotationQuaternion;
+        [this.xGizmo, this.yGizmo, this.zGizmo].forEach((gizmo) => {
+            if (gizmo) {
+                gizmo.customRotationQuaternion = customRotationQuaternion;
+            }
+        }, this);
+    }

```

Comment:
This block of code is duplicated several times: it's not possible to move it in the base `Gizmo` class?